resources/lang/en/admin/settings/general.php

```diff
@@ -75,6 +75,8 @@
     'label_logo_size'           => 'Square logos look best - will be displayed in the top right of each asset label. ',
     'laravel'                   => 'Laravel Version',
     'ldap'                      => 'LDAP',
+    'ldap_default_group'        => 'Default Permissions Group',
+    'ldap_default_group_info'   => 'Select a group to assign to newly synced users, remember that a user takes on the permissions of the group they are assigned.',
```

Comment:
`Select a group to assign to newly synced users, remember that a user takes on the permissions of the group they are assigned.`

should probably be 

`Select a group to assign to newly synced users. Remember that a user takes on the permissions of the group they are assigned.`

Reply:
fixed